tests/consensus_tests/test_partial_snapshot.py

```
@@ -0,0 +1,173 @@
+
+    return read_peer
+
+def bootstrap_peer(path: pathlib.Path, port: int):
+    path.mkdir()
+
+    config = {
+        "QDRANT__LOG_LEVEL": "debug,collection::common::file_utils=trace",
+        "QDRANT__FEATURE_FLAGS__USE_MUTABLE_ID_TRACKER_WITHOUT_ROCKSDB": "true",
+    }
+
+    uris, _, _ = start_cluster(path, 1, port_seed = port, extra_env = config)
```

Ruff (0.8.2)

`start_cluster` may be undefined, or defined from star imports

(F405)